### PR TITLE
Msm users frontend.go

### PR DESCRIPTION
**Pull request by manpreet85**
Added a rack2 to DBInventory

### CODE REVIEW

File: internal/services/frontend/DBInventory.go [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+package frontend
+
+import (
+	"sync"
+)
+
+type Tor struct {
+}
+type Pdu struct {
```
Comment (Jim3Things) [inline internal/services/frontend/DBInventory.go:17]: Please add a blank line between type declarations.

File: internal/services/frontend/DBInventory.go [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+			blades: make(map[int64]*blade),
+		}
+		dbInventory.Racks["rack2"] = &rack{ //
+
+			tor:    Tor{},
+			pdu:    Pdu{},
+			blades: make(map[int64]*blade),
+		}
+		dbInventory.Racks["rack1"].blades[1] = &blade{} //First rack added.
+		dbInventory.Racks["rack2"].blades[1] = &blade{}
```
Comment (Jim3Things) [inline internal/services/frontend/DBInventory.go:57]: Please add two blades per rack.

File: internal/services/frontend/doc.go [hunk 3]
```diff
@@ -104,10 +104,11 @@ operation code is invalid in some way.
 GET - /api/racks
 Get the list of all known racks.
 
-GET - /api/racks/{rackid}
+GET - /api/racks/{rackid}/
 Returns a single rack ID record.
 
 GET - /api/racks/{rack-id}/blades
+
```
Comment (Jim3Things) [inline internal/services/frontend/doc.go:111]: nestle the description next to the http request title line.

File: internal/services/frontend/DBInventory.go (hunk 1 above)
Comment (manpreet85) [reply internal/services/frontend/DBInventory.go:17]: > Please add a blank line between type declarations.

Completed

**Pull request merged**